wa/framework/execution.py

```diff
@@ -142,7 +142,6 @@ def start_job(self):
         job_output = init_job_output(self.run_output, self.current_job)
         self.current_job.set_output(job_output)
         self.update_job_state(self.current_job)
-        self.tm.start()
```

Comment:
the `stop()` should also be moved into the runner.

Reply:
Fixed.